config/locales/en.yml

```diff
@@ -21,3 +21,11 @@
 
 en:
   hello: "Hello world"
+
+  breakdown:
+    charge_categories:
```

Comment:
These might not be necessary; the changes were made directly in the sample data (sorry, there was a note on the waffle). I think we should hold off on locales for charge categories & similar until we speak with the users, since those were all generated in-house.